packages/salesforcedx-vscode-core/src/commands/forceRenameLightningComponent.ts

```diff
@@ -93,7 +99,10 @@ export class GetComponentName
   }
 }
 
-export async function inputGuard(sourceFsPath: string, newName: string): Promise<string> {
+export async function inputGuard(
+  sourceFsPath: string,
+  newName: string
+): Promise<string> {
```

Comment:
I noticed something here.  Is `newName` a string, or is it a `Promise<string>`?  Seems like it's just a string, and `inputGuard` should be changed to:
```
export async function inputGuard(
  sourceFsPath: string,
  newName: string
): string {
```

Reply:
nm...